training/platform-services/PLATFORM.md

```diff
@@ -0,0 +1,182 @@
+
+Please note that when deploying changes to the `production` environment (the live website), you need to **manually** deploy the application by doing the following:
+1. Go to [Concourse][concourse-ons]
+2. Navigate to the application
+3. Click on `production-ship-it`
+4. <div>Click on the <img src="images/concourse-build.png" title="Concourse Build" width=4% height=4%></div>
+5. Login to concourse as `dp` user (if you do not have the login, please ask anyone in the team)
+
+More information about Concourse can be found on their website - https://concourse-ci.org/docs.html 
+
+## Consul
+Visit the following to access Consul
+- DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://consul.develop.onsdigital.co.uk
+- PRODUCTION:&nbsp;&nbsp;&nbsp;https://consul.production.onsdigital.co.uk
+
+[<img src="images/consul-dev-services.png" title="Consul Develop Home" width=49.7% height=49.7%/>][consul-dev-services] [<img src="images/consul-dev-nodes.png" title="Consul Develop Nodes" width=49.7% height=49.7%/>][consul-dev-nodes]
+
+> Consul is a service mesh solution providing a full featured control plane with service discovery, configuration, and segmentation functionality. Each of these features can be used individually as needed, or they can be used together to build a full service mesh
+
+Consul is mainly used to check the health of applications which Consul retrieves by calling the health check endpoints of the application (`/health`). This information is then used to monitor cluster health, and it is used by the service discovery components to route traffic away from unhealthy hosts
+
+More information about Consul can be found on their website - https://www.consul.io/docs/intro
+
+## Nomad
+Visit the following to access Nomad
+- DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://nomad.develop.onsdigital.co.uk
+- PRODUCTION:&nbsp;&nbsp;&nbsp;https://nomad.production.onsdigital.co.uk
+
+[<img src="images/nomad-dev-home.png" title="Nomad Develop Home" width=49.7% height=49.7%/>][nomad-dev-home] [<img src="images/nomad-dev-job.png" title="Nomad Develop Job Page" width=49.7% height=49.7%/>][nomad-dev-home]
+
+> Nomad is a simple and flexible workload orchestrator to deploy and manage containers and non-containerized applications across on-prem and clouds at scale
+
+Nomad enables us to deploy our applications by bundling computing resources of [AWS][AWS]. In Nomad, each application is considered as a [job][nomad-job] which can have multiple [task groups][nomad-task-group] (`web` or `publishing`). The mapping of these tasks in a job is done using allocations, which is used to declare that a set of tasks in a job should be run on a particular node. Nomad would schedule these allocations which is detailed more [here][nomad-scheduling].   
```

Comment:
I don't think the term node has been explained in this context yet? so where you say 'job should be run on a particular node' maybe just an additional bit of context like 'a particular node within the AWS Auto-scaling Group (ASG)'

Or we could go into more detail that the nomad server is talking to its nomad clients, which each sit on a node in an ASG, and actually the allocations are assigned to a nomad client, which then manages the allocation for its lifecycle on that node